std_msgs/Header header
sensor_msgs/PointCloud2 cloud
float32 begin_angle
float32 end_angle
duration scan_time
